docs/DeveloperGuide.md

```diff
@@ -300,38 +300,34 @@ It will also include settings to create a custom look for the application.
+| `* *`    | student living in dorm                    | update when my friends are free                                      | I ensure that the free times stored are accurate                                                |
+| `* *`    | new user of this application              | view allowed commands when the application launches                  | I am aware of what functions I can use in the application                                       |
+| `* *`    | user exploring this application           | get autocomplete when typing commands                                | I can quickly give my commands                                                                  |
+| `* *`    | student living in dorm with many contacts | search a contact by name                                             | I can quickly find details of my dorm mates                                                     |
+| `* *`    | student living in dorm with many contacts | filter contacts by dorm room number                                  | I can find where are my dorm mates                                                              |
+| `* *`    | student living in dorm with many contacts | search a contact by birthday                                         | I know whose birthday is in which month                                                         |
+| `* *`    | student living in dorm                    | add a profile picture for each contact                               | I can recognise and identify the contact person                                                 |
+| `* *`    | student living in dorm                    | add telegram handle for each contact                                 | I can contact them on telegram as an alternative to their phone number                          |
+| `*`      | student living in dorm                    | sync room updated room numbers across multiple users (decentralised) | I can just update my room number and not worry about other user’s room numbers                  |
+| `*`      | student living in dorm who enjoys events  | add upcoming events                                                  | I can remember what events are coming up                                                        |
+| `*`      | student living in dorm who enjoys events  | view upcoming events                                                 | I can plan for them accordingly                                                                 |
+| `*`      | student living in dorm who enjoys events  | view upcoming birthdays                                              | I can plan a celebration and wish them happy birthday                                           |
+| `*`      | new user of this application              | type help to get a list of all commands and how to use them          | I can find the command I want to use                                                            |
```

Comment:
Will "type help" be too UI-specific? I remember there is a part of requirement specification that mentions UI-specific details shouldn't be included, not sure whether it is referring to user stories.

Reply:
updated